libs/agno/agno/tools/mcp.py

```diff
@@ -165,20 +165,23 @@ async def __aenter__(self) -> "MCPTools":
             if "url" not in sse_params:
                 sse_params["url"] = self.url
             self._context = sse_client(**sse_params)
+            client_timeout = max(self.timeout_seconds, sse_params.get("timeout", self.timeout_seconds))
```

Comment:
should it not be `min`? I'd think the shorter timeout should hit first?

Reply:
rationale was to favor the value inducing less errors. But you're right I think, the min would be the expected one for me too, will adjust!